client/dashboard/sites/settings-legacy/summary.tsx

```diff
@@ -0,0 +1,35 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { institution } from '@wordpress/icons';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { canUpdateLegacyFeatures } from '../../utils/site-features';
+import type { Site, SiteSettings } from '../../data/types';
+import type { Density } from '@automattic/components/src/summary-button/types';
+
+export default function LegacySummary( {
+	site,
+	settings,
+	density,
+}: {
+	site: Site;
+	settings: SiteSettings;
+	density?: Density;
+} ) {
+	if ( ! canUpdateLegacyFeatures( site ) ) {
+		return null;
+	}
+
+	return (
+		<RouterLinkSummaryButton
+			to={ `/sites/${ site.slug }/settings/legacy` }
+			title={ __( 'Control your legacy' ) }
+			density={ density }
+			decoration={ <Icon icon={ institution } /> }
+			badges={
+				settings.wpcom_locked_mode
+					? [ { text: __( 'Site locked' ), intent: 'success' as const } ]
```

Comment:
I don't think this should be `success`; we're preventing posts / pages from being edited. I think it should be `warning` or maybe `info`.